Appl/FileMgrs2/CommonIS/CArt/mkrNDFolderDirUp.ui

```diff
@@ -1,306 +1,72 @@
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0x00, 0x00, 0x00 >
-		RGBValue < 0xf7, 0xf7, 0xe6 >
-		RGBValue < 0xf3, 0xf3, 0xaa >
-		RGBValue < 0x48, 0x48, 0x31 >
-		RGBValue < 0xc2, 0xc2, 0x81 >
-		RGBValue < 0xb0, 0xb0, 0x70 >
-		RGBValue < 0x18, 0x18, 0x8f >
-		RGBValue < 0xff, 0xff, 0xff >
+    color = color4;

```

Comment:
Is the icon looking perfect in 16 color mode as well? Otherwise this should state color8 here, because it used 8bit color space.